utils/doclint/generateDotnetApi.js

```diff
@@ -316,6 +326,8 @@ function generateNameDefault(member, name, t, parent) {
     if (member.kind === 'method' || member.kind === 'property') {
       // this should be easy to name... let's call it the same as the argument (eternal optimist)
       let probableName = `${parent.name}${translateMemberName(``, name, null)}`;
+      if (specialNameResolutionMap.has(probableName))
```

Comment:
Like I mentioned in the chat it'd be nice if the list had only exceptions and most of the names would be generated in accord with some common rule (like ParamName or ClassParamName etc). While `SelectOption` looks like an exception Position could probably be derived by a common rule. We'll need this mapping at least in one more place - playwright.dev generator

Reply:
Yeah, that's a valid point. 

I've gone and improved this so the names now being generated follow a _logical_ rule. The only "special case" I had to add was for names that are _too generic_, like `value`, which gets translated to `SelectOptionValue`. But still, I'm much happier with this iteration.